src/loaders/express.ts

```diff
@@ -93,8 +94,8 @@ const loadUser = (app: Express): void => {
 const loadRouter = (app: Express): void => {
   app.use("/", homeRouter);
   app.use("/", authRouter);
-  app.use("/users", userRouter);
-  app.use("/posts", postRouter, likeRouter, retweetRouter);
+  app.use("/users", userRouter, followRouter);
+  app.use("/posts", postRouter, likeRouter, retweetRouter, followRouter);
```

Comment:
同じルーティングを複数のURLで受けるのは良くないので、posts側は不要そうです！

```suggestion
  app.use("/posts", postRouter, likeRouter, retweetRouter);
```